node/src/blockchain/blockchain_interface.rs

```diff
@@ -1201,17 +1202,17 @@ mod tests {
             Chain::EthMainnet,
         );
 
-        assert_gas_limit_is_not_under(subject, 55000, 65000)
+        assert_gas_limit_is_between(subject, 55000, 65000)
     }
 
-    fn assert_gas_limit_is_not_under<T: Transport + Debug + 'static>(
+    fn assert_gas_limit_is_between<T: Transport + Debug + 'static>(
         subject: BlockchainInterfaceNonClandestine<T>,
         not_under_this_value: u64,
         not_above_this_value: u64,
     ) {
         let sign_transaction_params_arc = Arc::new(Mutex::new(vec![]));
         let consuming_wallet_secret_raw_bytes = b"my-wallet";
-        let send_transaction_tools = &SendTransactionToolWrapperMock::default()
+        let send_transaction_tools = &SendTransactionToolsWrapperMock::default()
             .sign_transaction_params(&sign_transaction_params_arc)
             //I don't want to set up all the mocks - I want see just the params coming in
             .sign_transaction_result(Err(Web3Error::Internal));
```

Comment:
I can't mark the specific lines I'm after, but they're 1230 and 1231 at the time I'm marking: they're the ones with > and < comparisons in them about a dozen lines down.

This isn't your fault, but given the names of those parameters, it looks as though the > and < should be >= and <=. If you agree, change them.